app/controllers/topics_controller.rb

```diff
@@ -757,7 +757,13 @@ def invite_group
 
     if topic.private_message?
       guardian.ensure_can_invite_group_to_private_message!(group, topic)
-      topic.invite_group(current_user, group)
+      should_notify =
+        if params[:should_notify].blank?
+          true
+        else
+          params[:should_notify] == "true"
```

Comment:
```suggestion
          params[:should_notify].to_s == "true"
```
Params might be an actual boolean, if the API request is sent in JSON mode